README.md

```diff
@@ -11,7 +12,7 @@ This library provides the following:
 
 ---
 
-**NOTE**: You will need [Xcode 11.4](https://apps.apple.com/gb/app/xcode/id497799835?mt=12) or [Swift 5.2](https://swift.org/download/#swift-52) to try out `AsyncHTTPClient`.
+**NOTE**: You will need [Xcode 13.2](https://apps.apple.com/gb/app/xcode/id497799835?mt=12) or [Swift 5.5.2](https://swift.org/download/#swift-552) to try out `AsyncHTTPClient`s new async/await APIs.
```

Comment:
Should this note be added in addition to the previous one?

Reply:
I think this is now fairly old and we can remove it. I think (or at least hope) that every new user is already on an Xcode/Swift version that came out in the last 2 years.

Reply:
True, and we're about to drop support for 5.2 and 5.3 anyway!